source/AI.cpp

```diff
@@ -1348,6 +1354,18 @@ bool AI::FollowOrders(Ship &ship, Command &command) const
 		else
 			command.SetTurn(TurnToward(ship, TargetAim(ship)));
 	}
+	else if(type == Orders::COLLECT)
+	{
+		if(DoHarvesting(ship, command))
```

Comment:
I'll be opening a PR to change this to account for staying around the flagship when there is not anymore cargo space.